python/revng/cli/_commands/fetch_debuginfo/common.py

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env python3
```

Comment:
Drop shebang from all files that are not supposed to be run directly (i.e., have no `main`).